Step+XMLAdditions.h

```diff
@@ -0,0 +1,18 @@
+//
+//  Step+XMLAdditions.h
+//  DemoMonkey
+//
+//  Created by Nate Chandler on 3/15/15.
+//
+//
+
+#import "Step.h"
+
+extern NSString * __nonnull const StepXMLAdditionsKeyStep;
```

Comment:
Blargh. Use of these modern nullability thingies requires Xcode 6.3, doesn't it? This won't compile for me with 6.2